### ISSUE TITLE
VT - All SimpleReport Messages Failing Due to Errors in OBX Segment

### ISSUE THREAD

**Issue by Adrian-Brewster** [CDCgov/prime-reportstream#3407]
**Describe the bug**
 Vermont is experiencing a new issue in the feed as of the 3:15p file today. All messages in the batch failed due to this error in the OBX (result) segment. Was additional untested new code deployed to production today between 1:15 and 3:15 pm, and can it be rolled back as soon as possible? 

**Impact**
All SimpleReport issues failing

**Examples**
OBX|1 segment in failed messages:
 
OBX|1|CW|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.68||260415000|||N^Normal (applies to non-numeric results)^HL70078^^^^2.7|||F|||……
 
OBX|1 segment in previous messages (approved and expected format):
 
OBX|1|CWE|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.68||260415000^Not detected^SCT|||N^Normal (applies to non-numeric results)^HL70078^^^^2.7|||F|||……

**Comment by Adrian-Brewster** [comment]
Changes rolled back.  Issue was related to truncation issue.

**Comment by Adrian-Brewster** [comment]
Fix is in.  Waiting on approval for test message @jorg3lopez sent.

**Comment by anshulkumar-usds** [comment]
VT validated file.
